components/nds-forms/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@nice-digital/nds-forms",
   "version": "1.0.23-alpha.0",
-  "description": "Form components for the NICE Design System",
+  "description": "DEPRECATED! Use @nice-digital/nds-input and @nice-digital/nds-textarea",
```

Comment:
can we run `npm deprecate` once published too? https://docs.npmjs.com/deprecating-and-undeprecating-packages-or-package-versions